src/storage/src/hummock/local_version_manager.rs

```diff
@@ -327,37 +327,34 @@ impl LocalVersionManager {
         true
     }
 
-    /// Waits until the local hummock version contains the epoch. We will wait for different epochs
-    /// according to `HummockReadEpoch`'s type
-    pub async fn wait_epoch(&self, wait_epoch: HummockReadEpoch) -> HummockResult<()> {
-        if wait_epoch.get_epoch() == HummockEpoch::MAX {
+    /// Waits until the local hummock version contains the epoch. If `wait_epoch` is `Current`,
+    /// we will only check whether it is le `max_sync_epoch` and won't wait.
+    pub async fn try_wait_epoch(&self, wait_epoch: HummockReadEpoch) -> HummockResult<()> {
+        let wait_epoch = match wait_epoch {
+            HummockReadEpoch::Committed(epoch) => epoch,
+            HummockReadEpoch::Current(epoch) => {
+                assert!(
+                    epoch <= self.local_version.read().get_max_sync_epoch()
+                        && epoch != HummockEpoch::MAX
+                );
```

Comment:
I think we should print out the epoch in assertion to make debugging easier in case the assertion fails. We can also revisit the epoch-related code path to see how we can improve logging. We can do it in a separate PR.